panels/configspec.ini

```diff
@@ -16,6 +16,21 @@ panel_rooms = string_list(default=list("panels_1", "panels_2", "panels_3", "pane
 # deprecated "peglegs_email_signature" from the main repo.
 panels_email_signature = string(default=" - MAGFest Panels Team")
 
+# A list of social media fields collected from panelists.
+social_media = string_list(default=list("Facebook", "Twitter", "Instagram", "LinkedIn"))
```

Comment:
Yeah, I'll add a comment explaining the relationship. The `[social_media_urls]` and `[social_media_placeholders]` are nice to have, but not really necessary for the operation of the actual `SocialMediaMixin()` class.